### PR TITLE
Account trips a11y

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR improves the accessibility of the trip settings when saving/editing a trip.

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Layout is a little off for days where trips are not available, but pending that fix everything else looks good to me!

File: lib/components/user/monitored-trip/trip-basics-pane.tsx [hunk 1]
```diff
@@ -208,32 +253,32 @@ class TripBasicsPane extends Component<TripBasicsProps> {
...
+                      id={day}
+                      name={day}
+                      type="checkbox"
+                    />
+                    <label htmlFor={day}>
+                      <InvisibleA11yLabel>
+                        <FormattedDayOfWeek day={day} />
+                      </InvisibleA11yLabel>
+                      <span aria-hidden>
+                        {/* The abbreviated text is visual. Screen readers  */}
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/trip-basics-pane.tsx]: Thanks. Probably text I deleted and forgot to rephrase. Fixed in https://github.com/opentripplanner/otp-react-redux/pull/784/commits/161c2b760b58002a19e11adb7b8cf2526b32547c.

File: lib/components/user/monitored-trip/trip-basics-pane.tsx [hunk 2]
```diff
@@ -53,23 +55,61 @@ type TripBasicsProps =
...
+    display: none;
+  }
+
+  input,
+  .glyphicon::before {
+    margin-top: 1.8em;
+  }
+
+  /* Check boxes for disabled days are replaced with the cross mark. */
+  input[disabled] {
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/trip-basics-pane.tsx:100]: Good catch, fixed in https://github.com/opentripplanner/otp-react-redux/pull/784/commits/161c2b760b58002a19e11adb7b8cf2526b32547c.

**Review by amy-corson-ibigroup — APPROVED** [review]

**Comment by demory** [comment]
I'm noticing that hitting 'return' submits the form in places where I wasn't expecting it -- e.g. trying to toggle a checkbox or open a dropdown. Is that the indended behavior?

I know that other keys are the "official" ones for those actions (space for checkbox, down for dropdowns), but wondering if some users might expect return to work as an alternate rather than submit the form?

**Comment by binh-dam-ibigroup** [comment]
> I'm noticing that hitting 'return' submits the form in places where I wasn't expecting it -- e.g. trying to toggle a checkbox or open a dropdown. Is that the indended behavior?
> 
> I know that other keys are the "official" ones for those actions (space for checkbox, down for dropdowns), but wondering if some users might expect return to work as an alternate rather than submit the form?

Yeah, `Enter` officially submits the form, so in a form, to activate other controls you use `Space`.

**Comment by binh-dam-ibigroup** [comment]
> Yeah, `Enter` officially submits the form, so in a form, to activate other controls you use `Space`.

If you add a checkbox to [this form example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#try_it), you will see the behavior you mention.

**Review by demory — APPROVED** [review]

**Pull request merged**